### ISSUE TITLE
Multiple Workspaces

### ISSUE THREAD

**Issue by cp1797** [silverwind/droppy#321]
Is it possible to have multiple workspaces for each user? (So that each one has a private storage sorta thing)

**Comment by silverwind** [comment]
No, that's not possible yet. I'd imagine it could be done by adding a `home` property to users that can be configured by privileged users in the options. After that, the APIs have to be adjusted to lock that user to the path.

**Comment by cp1797** [comment]
Perhaps in the future then. Label this as a __feature__  maybe ?
